docs/changes/utils/13004.other.rst

```diff
@@ -0,0 +1 @@
+Re-added maia.usno.navy.mil to the list of IERS A data download sources.
```

Comment:
```suggestion
Reverted ``astropy.utils.iers.iers.IERS_A_URL`` to ``maia.usno.navy.mil`` domain instead of NASA FTP.
```

Reply:
Alright... does GitHub lack the magic "squash commits" button that I have in GitLab? If so, I am sad.

Reply:
Squashed the manual way